modules/ROOT/pages/importword.adoc

```diff
@@ -0,0 +1,60 @@
+
+= Import from Word plugin
+:navtitle: Import from Word
+:description: Provides a way to import .docx (Word documents) or .dotx (Word templates) files into the editor.
+:description_short: Import .docx or .dotx files into the editor.
+:keywords: plugin, importword, import from word
+:pluginname: Import from Word
+:plugincode: importword
+:plugincategory: premium
+
+include::partial$misc/admon-paid-addon-pricing.adoc[]
```

Comment:
@kemister85 I think we need something like this, which I picked up from AI assistant docs.

`NOTE: This plugin is only available as a link:{aipricingurl}/[paid add-on] to link:{pricingpage}/[a TinyMCE subscription].`

Reply:
Updating this as part of the last commit, resolving this message.